rosdep/python.yaml

```
@@ -9947,6 +9947,19 @@ python3-yolov5:
   ubuntu:
     pip:
       packages: [yolov5]
+python3-yourdfpy-pip:
+  debian:
+    pip:
+      packages: [yourdfpy]
+  fedora:
+    pip:
+      packages: [yourdfpy]
+  osx:
+    pip:
+      packages: [yourdfpy]
+  ubuntu:
+    pip:
+      packages: [yourdfpy]
```

This can be simplified to cover all distros rather than a separate rule for each.

```suggestion
python3-yourdfpy-pip:
  '*':
    pip:
      packages: [yourdfpy]
```